src/test/resources/expected_helm_output/confluence/output.yaml

```diff
@@ -60,6 +106,70 @@ spec:
     app.kubernetes.io/name: confluence
     app.kubernetes.io/instance: unittest-confluence
 ---
+# Source: confluence/templates/statefulset-synchrony.yaml
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: unittest-confluence-synchrony
+  labels:
+    helm.sh/chart: confluence-0.9.0
```

Comment:
The `0.9.0` strings need to be updated to `0.10.0`, currently, the builds are failing.

- [ ] update version to fix the build